technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,29 @@
+# Performance Testing of TEAMMATES
+
+This report gives a detailed explanation of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes our proposed solution.
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [A brief overview of the Proposed Solution](#Overview-of-Solution)
+* [Tools considered for Performance Testing](#Tools-considered-for-Performance-Testing)
+* [Reasons for using JMeter](#Reasons-for-using-JMeter)
```

Comment:
I think this can go under the previous section

Reply:
Hmm as a subheading?

Reply:
Sure, but there should be more than 1 subheading in that case.
And this one can be `Reasons for choosing JMeter`